src/Handler.php

```diff
@@ -318,6 +340,19 @@ protected function getIncludesDefault() {
       'update.php',
       'web.config'
     ];
+
+    // Version specific variations.
+    switch ($version) {
+      case '8.0':
+      case '8.1':
+      case '8.2':
+        $common[] = '.eslintrc';
+        $common = array_diff($common, ['.eslintrc.json']);
```

Comment:
I would just remove .eslintrc.json from the initial array. Then add it in `default:` so that old versions doesn't have to "know" about future files.